packages/sheets/src/commands/mutations/set-worksheet-col-width.mutation.ts

```diff
@@ -56,6 +68,34 @@ export const SetWorksheetColWidthMutationFactory = (
     };
 };
 
+export const SetWorksheetColAutoWidthMutationFactory = (
+    params: ISetWorksheetColWidthMutationParams,
+    worksheet: Worksheet
+): ISetWorksheetColWidthMutationParams => {
+    const { unitId, subUnitId, ranges } = params;
+    const colWidthObj: IObjectArrayPrimitiveType<Nullable<number>> = {};
+    const manager = worksheet.getColumnManager();
+
+    for (let i = 0; i < ranges.length; i++) {
+        const range = ranges[i];
+        for (let j = range.startColumn; j < range.endColumn + 1; j++) {
+            const col = manager.getColumnOrCreate(j);
```

Comment:
Should not create?

Reply:
The function name "getColumnOrCreate" is not good; 

it should actually be getColumnDataWithDefaultValue.

Reply:
Understood. Maybe we should change that name.